CHANGELOG.asciidoc

```diff
@@ -7,96 +7,153 @@
 // Template, add newest changes here
 
 === Beats version HEAD
-https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
+https://github.com/elastic/beats/compare/1035569addc4a3b29ffa14f8a08c27c1ace16ef9...6.6[Check the HEAD diff]
```

Comment:
I was wondering why we were using a commit hash here instead of a tag, but I don't think we use one.

Reply:
we will change it to the tag v6.6.0 once we have one. I'm just using the hash for now so I can easily check for changes between build candidates. This is the build hash of the current BC.